instance/tests/models/test_openedx_appserver.py

```diff
@@ -28,7 +28,7 @@
 import requests
 import responses
 import yaml
-from ddt import ddt, data
+from ddt import ddt, data, unpack
```

Comment:
Just a nit: are we actually using `unpack` anywhere?

Reply:
I tried to :stuck_out_tongue:, in the end I needed a setting override and created a new test for it.
I will update the PR and remove this.